Auto Testing multiple charts in specified folder

MVladislav opened the pull request:
Implemented script from here: https://github.com/datreeio/helm-datree#testing-multiple-charts
for auto search all helm project under an specified folder, to not need to add for every single helm project an own config runtime.

Also added the line:
` echo "**Source path:** ${1}" >>"$GITHUB_STEP_SUMMARY"`
to know which project was checked when multi helm project folders will be checked.

All other processes should be work as before, only helm is changed.

Commit:
refactor: extend testing multi charts
- see info from here:
  https://github.com/datreeio/helm-datree#testing-multiple-charts
- + additional improvements

Commit:
refactor: test fix exit code

Commit:
refactor: add source path to report result

MVladislav committed:
refactor: add path info for last updates

helm path will be auto search in specified folder for projects

hadar-co commented on entrypoint.sh:
```diff
@@ -8,104 +8,130 @@ isKustomization="$INPUT_ISKUSTOMIZATION"
 
 printf "datree version: "
-datree version
+if [ "$isHelmChart" = "true" ]; then
```
since the helm plugin runs the Datree CLI, the interesting version here is that of the CLI and not the plugin. Please leave as it was:)

hadar-co commented on entrypoint.sh:
```diff
@@ -8,104 +8,130 @@ isKustomization="$INPUT_ISKUSTOMIZATION"
 else
-   echo "### 🥳 All rules passed successfully! 🥳" >> $GITHUB_STEP_SUMMARY
-   exit "$EXIT_STATUS"
+  datree test "$inputpath" $cliArguments
```
When using globs, this will not work. Please change `"$inputpath"` to `$inputpath` (remove the quotes). Something in the way Github parses/substitutes variables causes this to fail, which is why there were no quotes there in the first place.

hadar-co commented on entrypoint.sh:
```diff
@@ -8,104 +8,130 @@ isKustomization="$INPUT_ISKUSTOMIZATION"
+    echo ""
+  done < <(find "$inputpath" -type f -name 'Chart.y*ml')
+elif [ "$isKustomization" = "true" ]; then
+  datree kustomize test "$inputpath" $cliArguments -- $kustomizeArgs
```
Same as the comment below about `"$inputpath"` (remove quotes).

MVladislav replied:
alright, will be reverted.

Do you know if the same happens with the helm version? 
Here `$dir` is in quotes:
`helm datree test "$dir" $cliArguments -- $helmArgs`
i will leave this for now in quotes.

hadar-co replied:
when using helm, you shouldn't use globs since your PR now makes it recursive:) so no need to remove the quotes.
I'll make it clearer in the readme

hadar-co commented:
Hi @MVladislav thanks for this PR, looks great!
I left some comments, please have a look:)

Commit:
revert: print only cli datree version like origin
- https://github.com/datreeio/action-datree/pull/15#discussion_r992151584

Commit:
revert: run cli datree without " in $inputpath
- https://github.com/datreeio/action-datree/pull/15#discussion_r992151659
- https://github.com/datreeio/action-datree/pull/15#discussion_r992152783
